src/api_gen.jl

```diff
@@ -22,8 +22,18 @@ const typemap = @compat Dict{Symbol,Type}(
     :Integer => Integer,
     :Boolean => Bool,
     :String => Union(ByteString, Vector{Uint8}),
+    :ArrayOf_Integer => Array{Int, 1}
 )
 
+function maptype(s::Symbol)
```

Comment:
Nice, eventually we should build container types from the element types, but this is a good start :+1: